.github/workflows/merge-main.yml

```diff
@@ -201,58 +180,42 @@ jobs:
   deploy-test:
     name: TEST Deployment
     needs:
-      - cleanup-ghcr
       - codeql
       - sonarcloud
       - release
-    runs-on: ubuntu-latest
-    environment:
-      name: test
+    environment: test
+    runs-on: ubuntu-22.04
     env:
       ZONE: test
       NR_SPAR_BACKEND_VERSION: ${{ needs.release.outputs.version }}
+    strategy:
+      matrix:
+        name: [ init, database, backend ]
+        include:
+          - name: init
+            file: common/openshift.init.yml
+            overwrite: false
+          - name: database
+            file: database/openshift.deploy.yml
+            overwrite: false
+          - name: backend
+            file: backend/openshift.deploy.yml
+            overwrite: true
+            parameters: -p MIN_REPLICAS=1 -p MAX_REPLICAS=2 -p BUILD=test-${{ env.NR_SPAR_BACKEND_VERSION }}
```

Comment:
Remove MIN_REPLICAS and MAX_REPLICAS to use the template defaults, which are 3 and 5.